src/server_manager/web_app/ui_components/outline-intro-step.ts

```diff
@@ -0,0 +1,334 @@
+      .card .tag {
+        font-weight: 500;
+        letter-spacing: 0.05em;
+        font-size: 10px;
+        text-transform: uppercase;
+      }
+      .card#digital-ocean .tag,
+      .card#digital-ocean .email,
+      .card p {
+        color: var(--medium-gray);
+      }
+      .card#manual-server .tag {
+        color: var(--manual-server-green);
+      }
+      .card#aws .tag {
+        color: var(--aws-orange);
+      }
+      .card#gcp .tag {
+        color: var(--gcp-blue);
+      }
+      .card-title {
+        font-size: 20px;
+        line-height: 32px;
+        flex: 2; /* for spacing */
+        color: var(--light-gray);
+      }
+      .card img {
+        margin-top: 11px;
+      }
+      #digital-ocean img {
+        height: 22px;
+        width: 22px;
+      }
+      .card .description {
+        letter-spacing: 0;
+        line-height: 24px;
+      }
+      .card .description ul {
+        margin-top: 0px;
+        padding: 0px 20px;
+      }
+      .card .description ul li {
+        margin-bottom: 8px;
+      }
+      .card-footer {
+        padding-top: 12px;
+        border-top: 1px solid var(--border-color);
+      }
+      .card-footer paper-button {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        letter-spacing: 0.75px;
+        background-color: inherit;
+        color: var(--light-gray);
+      }
+      #digital-ocean .description ul {
+        /* NOTE: this URL must be relative to the ui_components sub dir,
+           unlike our <img src="..."> attributes */
+        list-style-image: url("../images/check_white.svg");
+      }
+      #manual-server .description ul {
+        list-style-image: url("../images/check_green.svg");
+      }
+      #aws .description ul {
+        list-style-image: url("../images/check_orange.svg");
+      }
+      #gcp .description ul {
+        list-style-image: url("../images/check_blue.svg");
+      }
+      /* Reverse check icon for RTL languages */
+      :host(:dir(rtl)) #digital-ocean .description ul {
+        list-style-image: url("../images/check_white_rtl.svg");
+      }
+      :host(:dir(rtl)) #manual-server .description ul {
+        list-style-image: url("../images/check_green_rtl.svg");
+      }
+      :host(:dir(rtl)) #aws .description ul {
+        list-style-image: url("../images/check_orange_rtl.svg");
+      }
+      :host(:dir(rtl)) #gcp .description ul {
+        list-style-image: url("../images/check_blue_rtl.svg");
+      }
+    `
+    ];
+  }
+
+  render() {
+    return html`
+      <outline-step-view>
+        <span slot="step-title">${this.localize('setup-title')}</span>
+        <span slot="step-description">${this.localize('setup-description')}</span>
+  
+        <div class="container">
+          <div id="digital-ocean" class="card" @tap="${this.connectToDigitalOceanTapped}">
+            <div class="card-header">
+              ${
+        this.digitalOceanEmail ? html`<div class="email">${this.digitalOceanEmail}</div>` :
```

Comment:
This if-operator is really hard to read. Can you put that in a variable instead?
if (this.digitalOceanEmail) {
  cardHeader = ...
} else {
  cardHeader = ...
}

${cardHeader}

Reply:
That's a really good idea. Thanks.

I fixed some of the formatting as well.